src/Tools/IiifHelper.php

```diff
@@ -13,46 +13,50 @@
 class IiifHelper {
 
   use StringTranslationTrait;
-  
-  protected $remoteInfoJsonURL;
 
+  protected $publicUrl;
+  protected $identifier;
```

Comment:
This is what i don't like. I don't think the constructor needs the identifier. Forces a new Object instances everytime we need to call some method with a new identifier, too much overhead